---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[S. Adusumalli]], [[M. F. Mohd Omar]], [[R. Soong]], [[T. Benoukraf]]
collections: [[epigenetic coral]]
Link to zotero:: [Adusumalli et al. - 2015 - Methodological aspects of whole-genome bisulfite s.pdf](zotero://select/library/items/B3Z298GU)
Journal: Briefings in Bioinformatics
DOI: 10.1093/bib/bbu016
Tags: #üîç

---
```ad-note
title: Abstract
color:: 

The combination of DNA bisulfite treatment with high-throughput sequencing technologies has enabled investigation of genome-wide DNA methylation beyond CpG sites and CpG islands. These technologies have opened new avenues to understand the interplay between epigenetic events, chromatin plasticity and gene regulation. However, the processing, managing and mining of this huge volume of data require specialized computational tools and statistical methods that are yet to be standardized. Here, we describe a complete bisulfite sequencing analysis workflow, including recently developed programs, highlighting each of the crucial analysis steps required, i.e. sequencing quality control, reads alignment, methylation scoring, methylation heterogeneity assessment, genomic features annotation, data visualization and determination of differentially methylated cytosines. Moreover, we discuss the limitations of these technologies and considerations to perform suitable analyses.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-31 4:06 pm

*adusumalliMethodologicalAspectsWholegenome2015*
	DNA methylation is a biochemical event involved in a number of biological processes, including cellular proliferation and differentiation, pluripotency, development, genomic imprinting and oncogenesis [1]. This DNA modification manifests its effects by enhancing or reducing the affinity of protein‚ÄìDNA interactions [2‚Äì4]. This in turn modulates the recruitment of transcription factors, thereby influencing gene activation or repression depending on the role of the affected regulatory elements (promoter, enhancer, silencer or insulator) and/or transcription factors (activator or repressor) [5‚Äì7]. DNA methylation has been commonly studied at CpG dinucleotides, which are often found in clusters termed CpG islands (CGI). 
	
[Page1](zotero://open-pdf/library/items/B3Z298GU?page=1&a=7AJ8W6BF)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference: ^8b9227

*adusumalliMethodologicalAspectsWholegenome2015*
	A large number of methods are available for the assessment of DNA methylation [14], with bisulfite treatment being a central procedure to a majority. The treatment deaminates unmethylated cytosines to uracil (which is later converted into thymine in DNA), while methylated cytosines are protected 
	
[Page1](zotero://open-pdf/library/items/B3Z298GU?page=1&a=DI7NH25A)
	
	
#methodology, #epigenetic, #methylation
	
	
	reference:

*adusumalliMethodologicalAspectsWholegenome2015*
	from this conversion [15‚Äì17]. 
	
[Page2](zotero://open-pdf/library/items/B3Z298GU?page=2&a=W3G8I5R6)
	
	
#methodology, #epigenetic, #methylation
	
	
	reference:




%% end annotations %%

## Bibliography

Adusumalli, S., Mohd Omar, M. F., Soong, R., & Benoukraf, T. (2015). Methodological aspects of whole-genome bisulfite sequencing analysis. _Briefings in Bioinformatics_, _16_(3), 369‚Äì379. [https://doi.org/10.1093/bib/bbu016](https://doi.org/10.1093/bib/bbu016)

%% Import Date: 2023-11-01T14:36:19.459+01:00 %%
