---
Used: false
collections: "[[ITS2]]"
tipe: article
---
tipe: journalArticle
Creators: [[Elisa Banchi]], [[David Stankovic]], [[Fernando Fernández-Mendoza]], [[Fabrizia Gionechetti]], [[Alberto Pallavicini]], [[Lucia Muggia]]
collections: [[ITS2]]
Link to zotero:: [Banchi et al. - 2018 - ITS2 metabarcoding analysis complements lichen myc.pdf](zotero://select/library/items/T98XMTY7)
Journal: Mycological Progress
DOI: 10.1007/s11557-018-1415-4
Tags: #ITS2

---
```ad-note
title: Abstract
color:: 

Lichen thalli harbor complex fungal communities (mycobiomes) of species with divergent trophic and ecological strategies. The complexity and diversity of lichen mycobiomes are still largely unknown, despite surveys combining culture-based methods and high-throughput sequencing (HTS). The results of such surveys are strongly influenced by the barcode locus chosen, its sensitivity in discriminating taxa, and the depth to which public sequence repositories cover the phylogenetic spectrum of fungi. Here, we use HTS of the internal transcribed spacer 2 (ITS2) to assess the taxonomic composition and diversity of a wellcharacterized, alpine rock lichen community that includes thalli symptomatically infected by lichenicolous fungi as well as asymptomatic thalli. Taxa belonging to the order Chaetothyriales are the major components of the observed lichen mycobiomes. We predict sequences representative of lichenicolous fungi characterized morphologically and assess their asymptomatic presence in lichen thalli. We demonstrated the limitations of metabarcoding in fungi and show how the estimation of species diversity widely differs when ITS1 or ITS2 are used as barcode, and particularly biases the detection of Basidiomycota. The complementary analysis of both ITS1 and ITS2 loci is therefore required to reliably estimate the diversity of lichen mycobiomes.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:48 am

*banchiITS2MetabarcodingAnalysis2018*
	The fungal nuclear ribosomal ITS2 region was amplified with the forward primer ITS3 and the reverse primer ITS4 (White et al. 1990). The amplicons for HTS were obtained by performing two PCR amplifications. The first PCR amplification used the forward and the reverse primers ITS3 and ITS4 modified with GC rich universal tails on the 5′-end (Carew et al.2013). The 5 ′-end tail was identical to the tail applied on the 3′-end of the barcodes used in the second PCR. The first PCR reaction mix contained 3 μl DNA template (10–20 ng), 3 μl HotMasterMix (5PRIME, Fisher Scientific), 0.5 μl BSA 10× (Sigma-Aldrich), 0.75 μlEvaGreen™ 20× (Biotium), 0.5 μl forward primer ITS3 (10 μM), and 0.5 μl reverse primer ITS4 (10 μM) in a final volume of 15 μl. The PCR amplifications were performed with CFX 96™ PCR System (Bio-Rad) with the following cycling profile: 94 °C for 2 min and 30 cycles at 94 °C for 20 s, 55 °C for 20 s, 65 °C for 40 s followed by a final extension at 65 °C for 2 min. A negative control was used to verify the absence of non-specific amplification products along the whole amplification and sequencing process. The second PCR amplification (switch PCR) was required for the multiplex sequencing and the attachment of the barcodes. It used primers modified with an BA^ adaptor and a sample-specific 10-bp barcode to the 5′-end of the forward primer, and a P1 adaptor to the 5′-end of the reverse primer. The reaction was performed in a mix containing 5 μlofthe first PCR product, 20 μl HotMasterMix (5PRIME), 2.5 μl EvaGreen™ 20× (Biotium), 1.5 μl forward primer (10 μM), and 1.5 μl reverse primer (10 μM) in a final volume of 50 μl. 
	
[Page3](zotero://open-pdf/library/items/T98XMTY7?page=3&a=PX3NKSGU)
	
	
	
	reference:

*banchiITS2MetabarcodingAnalysis2018*
	PCR conditions were the same as for the first PCR amplification but were repeated for 8 cycles. All the amplicons were checked for their quality and size by agarose gel electrophoresis and normalized using Mag-Bind® Normalizer Kit (Omega Biotek). Product concentrations were checked with NanoDrop® 2000 (Thermo Fisher Scientific). The amplicons of the different samples were pooled together in equimolar amounts and the resulting barcoded library was measured with Qubit™ Fluorimeter (Thermo Fisher Scientific) and sequenced with an in-house Ion Torrent Personal Genome Machine™ (PGM, Thermo Fisher Scientific) using a 314™ chip (Thermo Fisher Scientific) for a maximum read length of 400 bp. 
	
[Page3](zotero://open-pdf/library/items/T98XMTY7?page=3&a=TDHEQYSU)
	
	
	
	reference:


%% end annotations %%

## Bibliography

Banchi, E., Stankovic, D., Fernández-Mendoza, F., Gionechetti, F., Pallavicini, A., & Muggia, L. (2018). ITS2 metabarcoding analysis complements lichen mycobiome diversity data. _Mycological Progress_, _17_(9), 1049–1066. [https://doi.org/10.1007/s11557-018-1415-4](https://doi.org/10.1007/s11557-018-1415-4)

%% Import Date: 2023-10-06T10:56:04.640+02:00 %%
