---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[Groves Dixon]], [[Mikhail Matz]]
collections: [[epigenetic coral]]
Link to zotero:: [Dixon and Matz - 2021 - Benchmarking DNA methylation assays in a reef‐buil.pdf](zotero://select/library/items/MTJ6MDC6)
Journal: Molecular Ecology Resources
DOI: 10.1111/1755-0998.13282
Tags: 

---
```ad-note
title: Abstract
color:: 

Interrogation of chromatin modifications, such as DNA methylation, has the potential to improve forecasting and conservation of marine ecosystems. The standard method for assaying DNA methylation (Whole Genome Bisulfite Sequencing), however, is currently too costly to apply at the scales required for ecological research. Here we evaluate different methods for measuring DNA methylation for ecological epigenetics. We compare Whole Genome Bisulfite Sequencing (WGBS) with Methylated CpG Binding Domain Sequencing (MBD-seq), and a modified version of MethylRAD we term methylation-dependent Restriction site-Associated DNA sequencing (mdRAD). We evaluate these three assays in measuring variation in methylation across the genome, between genotypes, and between polyp types in the reef-building coral Acropora millepora. We find that all three assays measure absolute methylation levels similarly for gene bodies (gbM), as well as exons and 1Kb windows with a minimum Pearson correlation 0.66. Differential gbM estimates were less correlated, but still concurrent across assays. We conclude that MBD-seq and mdRAD are reliable and costeffective alternatives to WGBS. The considerably lower sequencing effort required for mdRAD to produce comparable methylation estimates makes it particularly useful for ecological epigenetics.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-11 5:00 pm

*dixonBenchmarkingDNAMethylation2021*
	DNA methylation is currently the easiest to measure, and the best-studied (Hofmann 2017). 
	
[Page3](zotero://open-pdf/library/items/MTJ6MDC6?page=3&a=LHULJWEQ)
	
	
#introduction, #methodology, #methylation
	
	
	reference: ^16c8ff

*dixonBenchmarkingDNAMethylation2021*
	WGBS, considered the gold standard for measuring DNA methylation, works by chemical conversion of unmethylated cytosines to uracils. Following PCR amplification, these bases are read as thymines. 
	
[Page4](zotero://open-pdf/library/items/MTJ6MDC6?page=4&a=HT3C6KHJ)
	
	
#bisulfite, #introduction, #methodology, #methylation
	
	
	reference:




%% end annotations %%

## Bibliography

Dixon, G., & Matz, M. (2021). Benchmarking DNA methylation assays in a reef‐building coral. _Molecular Ecology Resources_, _21_(2), 464–477. [https://doi.org/10.1111/1755-0998.13282](https://doi.org/10.1111/1755-0998.13282)

%% Import Date: 2023-11-01T14:36:29.463+01:00 %%
