---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[Jose M. Eirin-Lopez]], [[Hollie M. Putnam]]
collections: [[epigenetic coral]]
Link to zotero:: [Eirin-Lopez and Putnam - 2018 - Marine Environmental Epigenetics.pdf](zotero://select/library/items/S6VSMJZF)
Journal: Annual Review of Marine Science
DOI: 10.1146/annurev-marine-010318-095114
Tags: #aquaculture, #conservation, #üìù, #üîç, #üé®, #acclimatization, #biomonitoring, #global climate change, #phenotypic plasticity, #transgenerational epigenetic inheritance

---
```ad-note
title: Abstract
color:: 

Marine organisms‚Äô persistence hinges on the capacity for acclimatization and adaptation to the myriad of interacting environmental stressors associated with global climate change. In this context, epigenetics‚Äîmechanisms that facilitate phenotypic variation through genotype‚Äìenvironment interactions‚Äîare of great interest ecologically and evolutionarily. Our comprehensive review of marine environmental epigenetics guides our recommendations of four key areas for future research: the dynamics of wash-in and wash-out of epigenetic effects, the mechanistic understanding of the interplay of different epigenetic marks and the interaction with the microbiome, the capacity for and mechanisms of transgenerational epigenetic inheritance, and the evolutionary implications of the interaction of genetic and epigenetic features. Emerging insights in marine environmental epigenetics can be applied to critical issues such as aquaculture, biomonitoring, and biological invasions, thereby improving our ability to explain and predict the responses of marine taxa to global climate change.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:43 am

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	the epigenome (Bollati & Baccarelli 2010). The growing field of environmental epigenetics focuses on elucidating the cause‚Äìeffect relationships among the exposure to these environmental stressors, the modifications in epigenetic states, and the subsequent changes in the phenotypes of organisms. 
	
[Page3](zotero://open-pdf/library/items/S6VSMJZF?page=3&a=LMXMAU6A)
	
	
#introduction, #epigenetic
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	If heritable epigenetic variations can underlie evolutionary change, then a broader concept of heredity, along with the recognition that natural selection may act on several different types of heritable variation (Darwinian and Lamarckian), is necessary ( Jablonka & Lamb 2002). 
	
[Page4](zotero://open-pdf/library/items/S6VSMJZF?page=4&a=REQJGG9B)
	
	
#introduction, #epigenetic
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Most commonly, it is cytosine that is methylated in animals (DoskoÀá cil & Àá Sorm 1962), specifically in a cytosine-phosphate-guanine (CpG) sequence. 
	
[Page11](zotero://open-pdf/library/items/S6VSMJZF?page=11&a=FKQFIH3R)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	The DNA methylation reaction involves the enzyme-assisted transfer of a methyl group from S-adenosylmethionine to the C5 position of cytosine. Several enzymes called DNA methyltransferases (DNMTs), including DNMT3a, DNMT3b, and DNMT1 ( Jeltsch 2002), play roles in establishing (DNMT3a and DNMT3b) and maintaining (DNMT1) methylation patterns within the genome. 
	
[Page11](zotero://open-pdf/library/items/S6VSMJZF?page=11&a=6GB75REK)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	DNA methylation is not permanent but may be reverted by both passive demethylation (loss of methylation during replication, returning cytosine to an unmethylated state) and active demethylation [through the activity of ten eleven translocation (TET) proteins] (Figures 1 and 2), which oxidize 5-methylcytosine to generate 5-hydroxymethylcytosine, 5-formylcytosine, and 5-carboxylcytosine (Ito et al. 2011). These intermediates can be further broken down through thymine-mediated excision and base excision repair to unmethylated cytosine. Additionally, there is a propensity for methylated cytosine to mutate to thymine through deamination, resulting in a base change, which base excision repair can subsequently correct. 
	
[Page11](zotero://open-pdf/library/items/S6VSMJZF?page=11&a=FFNTGLG9)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	The rates of active methylation and active and passive demethylation are not known for many species, particularly nonmodel organisms, where they constitute a knowledge gap (see the sidebar titled Future Challenges and Questions). 
	
[Page11](zotero://open-pdf/library/items/S6VSMJZF?page=11&a=FX98J95P)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	DNA methylation is posited as a dynamic system for cellular memory through the regulation of gene expression and thus the developmental or tissue-specific fate of the cell. This role is supported, for example, by the connection of DNA methylation with development, disease, and environmentally induced plasticity 
	
[Page11](zotero://open-pdf/library/items/S6VSMJZF?page=11&a=FJI4YMGG)
	
	‚ÄúDNA methylation is posited as a dynamic system for cellular memory through the regulation of gene expression and thus the developmental or tissue-specific fate of the cell. This role is supported, for example, by the connection of DNA methylation with development, disease, and environmentally induced plasticity‚Äù ([Eirin-Lopez and Putnam, 2018, p. 345](zotero://select/library/items/RRH4BYLA)) ([pdf](zotero://open-pdf/library/items/S6VSMJZF?page=11&annotation=FJI4YMGG))
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Histone proteins constitute key structural elements that facilitate the packing of the long eukaryotic DNA within the limited space of the cell nucleus and therefore modulate access to different regions in the genome. These proteins are small and very basic, forming histone octamers that associate with the DNA and give rise to the nucleosome, the fundamental subunit of chromatin (van Holde 1989) (Figure 2). 
	
[Page14](zotero://open-pdf/library/items/S6VSMJZF?page=14&a=MASZM6FY)
	
	
#introduction, #epigenetic, #histone
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Some of these variants have been identified in marine invertebrates, including H2A.X, H2A.Z, and macroH2A, which are involved in critical roles in the epigenetic regulation of DNA structure and metabolism (Gonz ÃÅ alez-Romero et al. 2012, Moosmann et al. 2011). 
	
[Page14](zotero://open-pdf/library/items/S6VSMJZF?page=14&a=KW2PDXBU)
	
	
#introduction, #epigenetic, #histone
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Histone proteins are also amenable to more than 10 types of posttranslational modifications at multiple conserved residues (Allis & Jenuwein 2016) (Figure 2). These modifications (notably acetylation, methylation, and phosphorylation) not only participate in the regulation of chromatin structure but also help recruit proteins and chromatin-remodeling complexes that influence transcription as well as many other DNA processes, such as repair, replication, and recombination (Bannister & Kouzarides 2011). Combined with histone variants, histone posttranslational modifications also play critical roles in chromatin metabolism and epigenetic memory (Feil & Fraga 2012), contributing to major evolutionary transitions in eukaryotes (Eirin-Lopez & Ausi  ÃÅ o 2009). 
	
[Page14](zotero://open-pdf/library/items/S6VSMJZF?page=14&a=N9LQFGWZ)
	
	
#introduction, #epigenetic, #histone
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	In order to be vertically transmitted to the offspring, epigenetic modifications need to be present in the germline and endure the extreme reorganization of chromatin during gametogenesis, especially in the case of males. Since nucleosomes are disassembled and histones are widely replaced by protamines in mature spermatozoa (Eirin-Lopez & Ausi  ÃÅ o 2009), the epigenetic contributions of sperm chromatin to embryo development have been considered highly limited (Figure 3). However, a small fraction of the sperm chromatin retains a nucleosome configuration, including loci of developmental importance, such as imprinted gene clusters, microRNA clusters, HOX gene clusters, and the promoters of stand-alone developmental transcription and signaling factors (Hammoud et al. 2009). While the mechanisms underlying such transmission are still obscure, an intriguing possibility is the potential regulatory role that noncoding RNAs (ncRNAs), which are transported in the chromatoid bodies of gametes, could play in germ cells before and after fertilization (Kimmins & Sassone-Corsi 2005). 
	
[Page15](zotero://open-pdf/library/items/S6VSMJZF?page=15&a=S4QFGB2A)
	
	
#introduction, #epigenetic, #transgenerational
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	The regulatory role of ncRNAs involves the formation of RNA structures that facilitate the recruitment of chromatin-modifying complexes into specific chromatin regions, determining and maintaining specific epigenetic states (Holoch & Moazed 2015) (Figure 2). It has also been suggested that various classes of RNAs could be transferred horizontally across neighboring cells through extracellular vesicles, promoting environmental signal transduction, as well as potentially altering the epigenome of neighboring cells by carrying RNA-binding proteins (i.e., chromatin modifiers and transcription factors riding ncRNAs) (Di Liegro et al. 2017). Interestingly, RNAs (ncRNA and mRNA) can be methylated just like DNA, and these epigenetic modifications in what is now defined as the epitranscriptome (Table 1) may be key in the regulation of ncRNAs (Esteller & Pandolfi 2017). 
	
[Page15](zotero://open-pdf/library/items/S6VSMJZF?page=15&a=YPTWHN96)
	
	
#introduction, #epigenetic, #ncrna
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	transposons, among other roles (Iwasaki et al. 2015). miRNAs are ubiquitous short RNA molecules (18‚Äì26 nucleotides) that potentially bind mRNA transcripts from up to 200 different genes and modulate their expression by inhibiting mRNA translation or enhancing mRNA decay (Ulitsky & Bartel 2013). The miRNA transcriptome has been explored in marine organisms such as cnidarians (Moran et al. 2013), platyhelminthes (Resch & Palakodeti 2012), molluscs (Rosani et al. 2016), and teleost fishes (Bizuayehu & Babiak 2014). The results obtained by these studies suggest that miRNAs play critical regulatory roles during development, including mediating stem cell function. In addition, genes encoding miRNAs are expressed at a high level during development but display low and noninducible expression levels in adult organisms. Finally, evolutionary analyses are consistent with an ancient origin for the miRNA biogenesis machinery. 
	
[Page16](zotero://open-pdf/library/items/S6VSMJZF?page=16&a=K6FYNV4B)
	
	
#introduction, #epigenetic, #ncrna
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Accordingly, gene regulation mediated by ncRNAs has been linked to development, sex determination, and reproduction in oysters (Yu et al. 2016) and teleost fishes (Bizuayehu et al. 2012a,b, 2015; Cochrane et al. 2011, Mennigen et al. 2013, Robledo et al. 2017). A role for ncRNAs was also proposed during environmental responses to stress in marine organisms, including responses to virus and bacterial infection in Atlantic salmon (Bolta  ÃÉ na et al. 2016) and rainbow trout (Paneru et al. 2016). miRNA participation during responses to thermal stress has also been identified in Atlantic cod, although a study of DNA methylation in the promoters of differentially expressed miRNAs had inconclusive results (Bizuayehu et al. 2015). Beyond marine species, the role of miRNAs during other responses, such as osmoregulation and reactions to oxidative stress, osmotic stress, and toxins, has been revealed in freshwater teleost fishes (Bizuayehu & Babiak 2014). 
	
[Page17](zotero://open-pdf/library/items/S6VSMJZF?page=17&a=ERKYI4JZ)
	
	
#introduction, #epigenetic, #ncrna
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Acclimatization studies have focused on phenotypic plasticity, or the capacity for an organism to display a variety of phenotypes as a function of the environment. This examination of phenotypic plasticity has primarily been at two scales: intragenerational plasticity (IGP) and transgenerational plasticity (TGP). IGP is generated by signals occurring within the organism‚Äôs life, while TGP is induced by signals set prior to fertilization that modulate the offspring reaction norms in different ways (Salinas & Munch 2012) (Figure 1). These processes have the potential to provide environmental legacy effects, which to date have not often been considered despite the growing acknowledgment of their importance in a time of intense and abrupt climate change (Torda et al. 2017). 
	
[Page19](zotero://open-pdf/library/items/S6VSMJZF?page=19&a=GTHP26NV)
	
	
#introduction, #transgenerational
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	This flexibility in phenotype may provide a buffer against rapid environmental change, but the uncertainty of the role of epigenetics and acclimatization in climate change response comes from the wash-in and wash-out nature of these effects (Burggren 2015). The details by which the type, magnitude, and duration of a signal and the life stage at which exposure occurs contribute to a dynamic or potentially canalized response are largely unknown (see the sidebar titled Future Challenges and Questions). A mode of IGP on the more enduring end of the spectrum is developmental acclimatization, where the signals from the developmental environment set a trajectory toward a phenotype in anticipation of such conditions later in life. For example, the thickness of the fur coat of the meadow vole (Microtus pennsylvanicus) is generated by maternal environmental sensing of day length, which is translated as a hormonal signal to the developing offspring that primes a thickcoat phenotype, which should be matched to cooler conditions later in life (Lee & Zucker 1988). If environmentally induced epigenetic imprinting acts in an analogous way to the hormonal signal, a developmental trajectory could be epigenetically set. 
	
[Page19](zotero://open-pdf/library/items/S6VSMJZF?page=19&a=AI4V5FAM)
	
	
#introduction, #epigenetic, #conservation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	A variety of marine fish display TGP in response to both ocean warming and acidification. In particular, a strong positive response to adult preconditioning to ocean acidification has been documented in reef clownfish (Amphiprion melanopus) in terms of survival, size traits, and metabolic rate and a suite of movement and swimming metrics (reviewed in Donelson et al. 2018), supporting the capacity for beneficial traits to emerge through TGP 
	
[Page20](zotero://open-pdf/library/items/S6VSMJZF?page=20&a=DM9WSKLS)
	
	
#introduction, #epigenetic, #transgenerational, #conservation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	In reefbuilding corals, there is evidence for the inducibility of DNA methylation in an environmentally sensitive brooding coral (Pocillopora damicornis) following exposure to ocean acidification (Putnam et al. 2016). This same species shows beneficial acclimatization of the offspring at the larval stage following adult exposure during brooding (Putnam & Gates 2015) and beneficial effects of adult exposure to low pH on the settlement, survivorship, and growth of larvae months after adult exposure (Putnam et al. 2018). 
	
[Page20](zotero://open-pdf/library/items/S6VSMJZF?page=20&a=5YFCZF2J)
	
	
#introduction, #epigenetic, #conservation
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	For instance, the microbiome may have a potential role in TGP, and this is exemplified by systems such as reef-building corals, where the microbiome (as constituents of the metaorganism) can enhance adaptive capacity and environmental stress tolerance (Webster & Reusch 2017). 
	
[Page24](zotero://open-pdf/library/items/S6VSMJZF?page=24&a=XNAZX9VH)
	
	
#introduction, #epigenetic, #conservation, #microbiome
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Emergent properties of the interaction between epigenetic mechanisms and dynamic microbial communities provide fertile grounds for rapid adaptation through nongenetic means (see the sidebar titled Future Challenges and Questions). 
	
[Page24](zotero://open-pdf/library/items/S6VSMJZF?page=24&a=39WXC6DH)
	
	
#introduction, #epigenetic, #conservation, #microbiome
	
	
	reference:

*eirin-lopezMarineEnvironmentalEpigenetics2019*
	Marine environmental epigenetics represents an emerging research frontier, with the potential to revolutionize our understanding of ecological and evolutionary dynamics at a critical climate change juncture, in an invaluable marine setting. 
	
[Page25](zotero://open-pdf/library/items/S6VSMJZF?page=25&a=SNB7KV6A)
	
	
#introduction, #epigenetic
	
	
	reference:








%% end annotations %%

## Bibliography

Eirin-Lopez, J. M., & Putnam, H. M. (2019). Marine Environmental Epigenetics. _Annual Review of Marine Science_, _11_(1), 335‚Äì368. [https://doi.org/10.1146/annurev-marine-010318-095114](https://doi.org/10.1146/annurev-marine-010318-095114)

%% Import Date: 2023-11-01T14:36:32.685+01:00 %%
