---
Used: false
collections: "[[epigenetic coral/histone epigenetic]]"
tipe: article
---
tipe: book
Creators: [[Dong Fang]], [[Junhong Han]]
collections: [[epigenetic coral/histone epigenetic]]
Link to zotero:: [Fang and Han - 2021 - Histone Mutations and Cancer.pdf](zotero://select/library/items/IB8PBDRB)
Journal: 
DOI: 
Tags: #üîç

---
```ad-note
title: Abstract
color:: 

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:48 am

*fangHistoneMutationsCancer2021*
	Histone methylation, which usually occurs at the lysine (K) residues of histone H3 and H4 by adding methyl groups, is one of the most important post-transcriptional modifications [4]. 
	
[Page9](zotero://open-pdf/library/items/IB8PBDRB?page=9&a=2XKAA6PT)
	
	
#introduction, #epigenetic, #histonemethylation
	
	
	reference:

*fangHistoneMutationsCancer2021*
	This methylation is catalyzed by the histone methyltransferase (HMT), which uses S-adenosyl methionine (SAM) as the substrate to transfer methyl groups onto the lysine residues of histones [5] 
	
[Page9](zotero://open-pdf/library/items/IB8PBDRB?page=9&a=DZEJITHB)
	
	
#introduction, #epigenetic, #histonemethylation
	
	
	reference:

*fangHistoneMutationsCancer2021*
	The lysine residues of histones can be mono-, di-, and trimethylated (me1, me2, and me3, respectively) to act as the active or repressive marks of gene expression (Fig. 1.1). In general, H3K4, H3K36, and H3K79 are considered as active marks which occupy the actively transcribed gene regions in chromatin. H3K9, H3K27, and H4K20 are known as repressive marks that are usually associated with the silenced gene expression and condensed chromatin [6]. It is also worth noting that these histone marks are not exclusive to each other in the chromatin. There are bivalent or trivalent domains with two or three histone marks in the same genomic loci, respectively [7‚Äì9]. Until the first demethylase, lysine-specific demethylase 1 (LSD1), was found to erase the methylation at H3K4, histone methylation was once believed to be irreversible [10]. 
	
[Page9](zotero://open-pdf/library/items/IB8PBDRB?page=9&a=BDBV4DBF)
	
	
#introduction, #epigenetic, #histonemethylation
	
	
	reference:

*fangHistoneMutationsCancer2021*
	Histone acetylation is regulated by the balance between histone acetyltransferases (HATs) and histone deacetylases (HDACs). Acetylation can reduce the positive charge of the lysine residues, which will then inhibit the binding between histone tails and negatively charged DNA, leaving the underlying DNA exposed [85]. Therefore, histone acetylation is usually considered as an active histone mark [86, 87]. Moreover, the acetylation and methylation in the same lysine residues can act as antagonists to inhibit each other, leading to the crosstalk between different histone marks [88]. 
	
[Page12](zotero://open-pdf/library/items/IB8PBDRB?page=12&a=B258AVTB)
	
	
#introduction, #epigenetic, #histoneacetylation
	
	
	reference:

*fangHistoneMutationsCancer2021*
	Phosphorylation, one of the most common posttranslational modifications, also occurs at serine and tyrosine residues of histone proteins [111]. Histone phosphorylation participates in a wide range of cellular processes, including gene expression, cell cycle regulation, DNA damage repair, and asymmetric cell division [112]. Two of the important methylation sites, H3K9 and H3K27, share the same subsequent serine residue that can be phosphorylated, forming a ‚ÄúKS‚Äù domain. This close position between these two modification sites leads to the hypothesis that H3 phosphorylation may alter, at least in part, the affinity of readers or writers of the contacted lysine residues [113]. This crosstalk between histone phosphorylation and methylation allows the spatiotemporal control of histone marks, leading to the complicated but precise regulation of chromatin structure [114]. 
	
[Page13](zotero://open-pdf/library/items/IB8PBDRB?page=13&a=UQDWDYR5)
	
	
#introduction, #epigenetic, #histonephosphorylation
	
	
	reference:

*fangHistoneMutationsCancer2021*
	Chromatin remodeling is the process of dynamic changes of chromatin structure. The high-order chromatin structures are uneven, whereas some regions are loosened and some regions are condensed. In the loosened regions of chromatin, the DNAs are at an ‚Äúopen‚Äù status and could be accessed by other proteins which control the gene expression, including transcription factors and RNA polymerases [128]. It‚Äôs hard for transcription-associated proteins to retrieve the condensed regions that are at a ‚Äúclose‚Äù status. Therefore, the gene expression of these regions will be repressed. Chromatin remodeling alters the ‚Äúclose‚Äù and ‚Äúopen‚Äù status of chromatin as a ‚Äúgate‚Äù to regulate the gene expression and cellular processes. 
	
[Page14](zotero://open-pdf/library/items/IB8PBDRB?page=14&a=6MJI8QP3)
	
	
#introduction, #epigenetic, #chromatinremodeling
	
	
	reference:

*fangHistoneMutationsCancer2021*
	Chromatin remodeling is carried out by chromatin remodeling complexes (Table 1.1) [129]. The ATP-dependent chromatin remodeling complexes use the energy of ATP hydrolysis to disrupt the binding between DNA and histones, leading to the changes of nucleosome position [130]. These remodeling complexes can be divided into four families, SWI/SNF, ISWI, CHD, and INO80, depending on the sequence and structure of ATPases in the complexes (Fig. 1.2). 
	
[Page14](zotero://open-pdf/library/items/IB8PBDRB?page=14&a=NLYV3G2N)
	
	
#introduction, #epigenetic, #chromatinremodeling
	
	
	reference:

*fangHistoneMutationsCancer2021*
	SWI/SNF, which exists in both eukaryotes and prokaryotes, is the earliest ATP-dependent chromatin remodeler found in yeast [131]. The core subunit of SWI/SNF is swi2/snf2 which possesses a DNA-stimulated ATPase activity and destabilizes histone-DNA interactions in reconstituted nucleosomes [132]. Moreover, the ATPases of SWI/SNF family contain a bromodomain which can recognize and bind to the acetylated histones [133] 
	
[Page14](zotero://open-pdf/library/items/IB8PBDRB?page=14&a=APC6Q4US)
	
	
#introduction, #epigenetic, #chromatinremodeling
	
	
	reference:

*fangHistoneMutationsCancer2021*
	ISWI, firstly found in Drosophila, presents as distinct variants in different species. The ISWI family proteins alter nucleosome positioning to facilitate chromatin condensation [134]. In addition, ISWI activates gene expression by directly facilitating transcription-regulator-mediated transcription [135]. The ATPase of ISWI family has a HAND-SANT-SLIDE domain in its C-terminal region which recognizes and binds to the lysine of histone H3 [136]. In vivo, ISWI is associated with DNA repair [137] and cell fate determination [138]. 
	
[Page14](zotero://open-pdf/library/items/IB8PBDRB?page=14&a=VH3PT8X9)
	
	
#introduction, #epigenetic, #chromatinremodeling
	
	
	reference:

*fangHistoneMutationsCancer2021*
	The ATPase of CHD complex contains a chromodomain [139, 140]. The CHD family is 
	
[Page14](zotero://open-pdf/library/items/IB8PBDRB?page=14&a=KEXWTUDX)
	
	
#introduction, #epigenetic, #chromatinremodeling
	
	
	reference:

*fangHistoneMutationsCancer2021*
	then divided into three subfamilies depending on the identities of their subunits. Chd1 and Chd2, which belong to the first subfamily, contain a DNA binding domain preferentially binding to AT-rich regions [141]. The second subfamily, including Chd3 and Chd4, has a pair of zincfinger-like domains in their N-terminal regions instead of C-terminal regions [142]. The last subfamily is distinguished by the additional motifs like SANT domain or BRK domain in the C-terminal region [143]. 
	
[Page16](zotero://open-pdf/library/items/IB8PBDRB?page=16&a=KIYRR3T3)
	
	
#introduction, #epigenetic, #chromatinremodeling
	
	
	reference:

*fangHistoneMutationsCancer2021*
	INO80 is enriched at the replication fork and DNA damage sites, including holiday junctions and H2A variants (H2A.X and H2A.Z) [144, 145]. SWR1 complex, one of the INO80 complexes, catalyzes the replacement of the canonical H2A-H2B with the H2AZ-H2B variants in vivo [146]. Ino80 and Swr1 complexes, two remodelers of INO80 family in yeast, function antagonistically during DNA double strand breaks, promoting or blocking cell cycle check point adaption through regulating the incorporation of histone H2A variants, respectively [147]. 
	
[Page16](zotero://open-pdf/library/items/IB8PBDRB?page=16&a=ZMRD83AL)
	
	
#introduction, #epigenetic, #chromatinremodeling
	
	
	reference:

*fangHistoneMutationsCancer2021*
	In addition to the direct control by proteins, histone modifications are regulated by non-coding RNAs. Small non-coding PIWI-interacting RNA (piRNA), mainly found in the mammalian germ cells and stem cells, is identified after siRNA and miRNA, with a length mostly around 29‚Äì30 nucleotides [159, 160]. One well-known function of piRNA is that it recognizes and silences the transposon stable elements in germ cells. The piRNA binds to PIWI protein to form a complex that targets the transcriptional site of a transposon by recognizing its primary transcript and then recruits histone H3K9 methyltransferase to induce H3K9me3 at this locus, leading to the chromatin condensation and gene repression [161]. 
	
[Page16](zotero://open-pdf/library/items/IB8PBDRB?page=16&a=RCU4HF2R)
	
	
#introduction, #epigenetic, #noncodingRNA
	
	
	reference:

*fangHistoneMutationsCancer2021*
	In addition, long non-coding RNA (lncRNA), which is longer than 200 base pairs, mediates the histone modifications to regulate chromatin status [162, 163]. 
	
[Page16](zotero://open-pdf/library/items/IB8PBDRB?page=16&a=6RX3I9KF)
	
	
#introduction, #chromatinremodeling, #noncodingRNA
	
	
	reference:


%% end annotations %%

## Bibliography

Fang, D., & Han, J. (Eds.). (2021). _Histone Mutations and Cancer_ (Vol. 1283). Springer Singapore. [https://doi.org/10.1007/978-981-15-8104-5](https://doi.org/10.1007/978-981-15-8104-5)

%% Import Date: 2023-10-06T10:56:30.471+02:00 %%
