---
Used: false
collections: "[[Coral symbiont association]]"
tipe: article
---
tipe: journalArticle
Creators: [[Benjamin C. C. Hume]], [[Edward G. Smith]], [[Maren Ziegler]], [[Hugh J. M. Warrington]], [[John A. Burt]], [[Todd C. LaJeunesse]], [[Joerg Wiedenmann]], [[Christian R. Voolstra]]
collections: [[Coral symbiont association]]
Link to zotero:: [Hume et al. - 2019 - SymPortal A novel analytical framework and platfo.pdf](zotero://select/library/items/WYVTD9P2)
Journal: Molecular Ecology Resources
DOI: 10.1111/1755-0998.13004
Tags: 

---
```ad-note
title: Abstract
color:: 

We present SymPortal (SymPortal.org), a novel analytical framework and platform for genetically resolving the algal symbionts of reef corals using next‐generation sequencing (NGS) data of the ITS2 rDNA. Although the ITS2 marker is widely used to genetically characterize taxa within the family Symbiodiniaceae (formerly the genus Symbiodinium), the multicopy nature of the marker complicates its use. Commonly, the intragenomic diversity resultant from this multicopy nature is collapsed by analytical approaches, thereby focusing on only the most abundant sequences. In contrast, SymPortal employs logic to identify within‐sample informative intragenomic sequences, which we have termed ‘defining intragenomic variants' (DIVs), to identify ITS2‐type profiles representative of putative Symbiodiniaceae taxa. By making use of this intragenomic ITS2 diversity, SymPortal is able to resolve genetic delineations using the ITS2 marker at a level that was previously only possible by using additional genetic markers. We demonstrate this by comparing this novel approach to the most commonly used alternative approach for NGS ITS2 data, the 97% similarity clustering to operational taxonomic units (OTUs). The SymPortal platform accepts NGS raw sequencing data as input to provide an easy‐to‐use, standardization‐enforced, and community‐driven framework that integrates with a database to gain resolving power with increased use. We consider that SymPortal, in conjunction with ongoing large‐scale sampling and sequencing efforts, should play an instrumental role in making future sampling efforts more comparable and in maximizing their efficacy in working towards the classification of the global Symbiodiniaceae diversity.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:48 am

*humeSymPortalNovelAnalytical2019*
	novel	analytical	 framework	 and	platform	 for	genetically	resolving	the	algal	symbionts	of	reef	corals	using	next-generation	sequencing	(NGS)	data	of	the	 ITS2	rDNA. 
	
[Page1](zotero://open-pdf/library/items/WYVTD9P2?page=1&a=7CEMYWWV)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	SymPortal	is	able	to	resolve	 genetic	delineations	 using	the	ITS2	marker	at	a	level	that	was	previously	only	possible	by	using	additional	 genetic	markers. 
	
[Page1](zotero://open-pdf/library/items/WYVTD9P2?page=1&a=6Z8E9898)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	One	important	 factor	that	contrib utes	to	reef	resilience	 is	the	genetic	 identity	 of	the	algal	symbi onts	harboured	 by	the	reef-building	 corals 
	
[Page2](zotero://open-pdf/library/items/WYVTD9P2?page=2&a=NNRCWRRJ)
	
	
#introduction, #Symbiodiniaceae, #clade, #reefresilience
	
	
	reference: ^cdde9e

*humeSymPortalNovelAnalytical2019*
	Effective	 characterization	 of	 these	algal	symbionts, 	family	Symbiodiniaceae 	(formerly 	genus	 Symbiodinium	containing 	Clades	A–I,	now	equivalent 	to	family	 Symbiodiniaceae,	currently	with	seven	named	genera;	LaJeunesse	 et	al.,	2018),	is	therefore	important. 
	
[Page2](zotero://open-pdf/library/items/WYVTD9P2?page=2&a=SAWCD58P)
	
	
#Symbiodiniaceae, #clade, #introuction
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	Despite	 the	multitude	 of	markers	 available	 for	assessing 	 Symbiodiniaceae	diversity,	the	internal	transcribed	spacer	2	(ITS2)	 of	the	rRNA	gene	shows	an	uninterrupted	 popularity	and	remains	 the	most	commonly	 used	marker 
	
[Page2](zotero://open-pdf/library/items/WYVTD9P2?page=2&a=GUF36I64)
	
	
#ITS2, #introduction
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	However,	 this	marker	 is	multicopy	 in	nature, 	 which	complicates 	its	use.	A	single	Symbiodiniaceae 	cell	usually	 contains	hundreds	to	thousands	 of	rRNA	gene	copies 
	
[Page2](zotero://open-pdf/library/items/WYVTD9P2?page=2&a=L62BGHD3)
	
	
#ITS2, #introduction
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	On	the	one	hand,	diversity 	of	 ITS2	sequences	can	come	from	sequence	variations	among	these	gene	copies,	 giving	rise	to	intrag enomic	diversity.	On	the	other	hand,	sample 	sequence	diversity	 may	also	be	due	to	hosts	associating	 with	multiple	 genotypes	of	 Symbiodiniaceae,	referred	to	as	intergenomic	diversity 
	
[Page2](zotero://open-pdf/library/items/WYVTD9P2?page=2&a=XRDSWRBP)
	
	
#ITS2, #introduction
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	Without	the	use	of	additional	 genetic	 markers,	differentiating	among	these	sources	 of	variation	 can	be	 challenging	(Thornhill	et	al.,	2007). 
	
[Page2](zotero://open-pdf/library/items/WYVTD9P2?page=2&a=K3654UJQ)
	
	
#ITS2, #introduction
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	Most	analyses 	successfully 	incorporated 	intragenomic 	theory, 	 leading	 to	the	core	understanding	 that	specific	 combinations	 of	 intragenomic	sequences	(commonly	known	as	ITS2	profiles)	could	 differentiate,	and	therefore	 define,	taxa	(often	referred	 to	as	socalled	‘types’;	LaJeunesse,	2002). 
	
[Page2](zotero://open-pdf/library/items/WYVTD9P2?page=2&a=AYDYITL3)
	
	
#ITS2, #introduction
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	We	employ	novel	logic	to	identify	within-sample	informative	 intragenomic	sequences, 	which	we	have	termed 	'defining 	intragenomic	variants'	(DIVs),	and	use	combinations	 of	these	DIVs	to	iden tify	 ITS2	type	profiles	 representative	 of	putative	 Symbiodiniaceae	 taxa.	This	approach	means	that	SymPortal	is	better	able	to	differentiate	between	intragenomic	and	intergenomic	sources	of	ITS2 variation	without	the	need	for	more	resolute	markers,	such	as	the	psbAncr. 
	
[Page3](zotero://open-pdf/library/items/WYVTD9P2?page=3&a=L6XKJA4G)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	Within	the	SymPor tal	 framework,	the	SymPor tal	 analysis	 works	 by	employing	the	usually	substantial	ITS2	intragenomic	diversity	harboured	within	every	Symbiodiniaceae	genome,	and	captured	by	NGS	 ITS2	sequencing,	to	resolve	between	genetically	differentiated	taxa	 (Figure	1).	The	analysis	 does	this	by	identifying	 specific	sets	of	‘de fining	intragenomic	[ITS2	sequence]	variants’	(DIVs)	that	represent 	 the	taxonomic	 unit	of	SymPortal,	 the	‘ITS2	type	profile’	 (a	term	de rived	from	the	‘ ITS2	types’	defined	by	‘ITS2	profiles’	 in	DGGE-based	 methodologies). 
	
[Page3](zotero://open-pdf/library/items/WYVTD9P2?page=3&a=GLYEP2RR)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	The	database	 with	which	the	SymPortal	 analysis	 is	integrated	 stores	sequencing	 information	 from	every	previously	 conducted	 SymPortal	 analysis.	 With	continued	 use,	the	SymPortal	 framework	will	therefore	accrue	a	catalog	of	ITS2	type	profiles	representative	of	identified	putative	Symbiodiniaceae	taxa. 
	
[Page3](zotero://open-pdf/library/items/WYVTD9P2?page=3&a=N7D9J5QN)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	Whilst	several	different	Symbiodiniaceae	taxa	may	be	harboured	 by	an	individual	coral,	commonly	only	a	single	Symbiodiniaceae	 taxon	 is	found	to	be	predominant 
	
[Page3](zotero://open-pdf/library/items/WYVTD9P2?page=3&a=JTUFDRN3)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	In	cases	where	multiple	 Symbiodiniaceae	are	harboured	by	a	single	coral,	these	taxa	are	usu ally	from	different 	genera. 	Nevertheless, 	although 	rare,	individual 	 corals	may	harbour	 multiple	 Symbiodiniaceae	 taxa	from	the	same	 genera.	SymPortal	must	therefore	 be	able	to	differentiate	 between	 intra-	and	intergenomic	 sources	of	ITS2	sequence	variants	returned	 from	sample	amplicon	libraries 
	
[Page3](zotero://open-pdf/library/items/WYVTD9P2?page=3&a=UEE42EUR)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	To	achieve	 this,	SymPortal	 applies	a	central	 principle:	the	probability	that	a	given	set	of	ITS2	sequences	found	in	 a	single	coral	sample	are	representative	of	a	single	Symbiodiniaceae	 genotype	increases	with	the	number	 of	samples	that	set	is	found	in	 (F ig ure	1c) . 
	
[Page3](zotero://open-pdf/library/items/WYVTD9P2?page=3&a=Z3T2LNT9)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	On	the	rare	occasions	 when	multiple	 taxa	belonging	 to	the	same	genus	are	found	within	a	single	sample,	 these	taxa	may	be	successfully	resolved	using	SymPortal	when	each	 of	them	is	present	in	other	samples	as	standalone	taxa. 
	
[Page3](zotero://open-pdf/library/items/WYVTD9P2?page=3&a=UCXK3S74)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	 
	
[Page4](zotero://open-pdf/library/items/WYVTD9P2?page=4&a=HQFKR9LT)
	
	
#symportal
	
	
	![[Documents/ImageArticles/image-4-x44-y38.png]] 
	
	reference:

*humeSymPortalNovelAnalytical2019*
	 
	
[Page5](zotero://open-pdf/library/items/WYVTD9P2?page=5&a=ZCYVPCNE)
	
	
#symportal
	
	
	![[Documents/ImageArticles/image-5-x39-y34.png]] 
	
	reference:

*humeSymPortalNovelAnalytical2019*
	When	a	set	of	 ITS2	sequences	 is	found	to	re-occur	 in	a	sufficient	 number	of	samples,	 each	of	these	sequences	 is	considered	to	be	a	DIV	and	these	DIVs	are	used	to	characterize	 an	ITS2 type	profile	(Figures	1	and	2).	If	for	any	given	sample	 a	re-occurring	 set	of	DIVs	cannot	 be	found,	 then	an	 ITS2	type	profile	will	be	assigned	 to	this	sample	 that	is	defined	 simply	 by	the	sample's	 most	 abundant ITS2	sequence. When	identifying	ITS2	type	profiles,	the	presence	as	well	as	the	 abundance	 of	characterizing	 DIVs	are	considered.	 For	each	set	of	 supported	 ITS2	sequences	that	is	found	in	the	algorithmic 	search,	 each	constituent 	 ITS2	sequence	is	tested	to	see	whether 	it	has	a	 multimodal	distribution	across	the	samples 	in	which	the	 ITS2	set	of	 sequences	was	found	(i.e.	whether	 there	are	two	or	more	distinctly 	 different	abundances	of	the	sequence). 
	
[Page6](zotero://open-pdf/library/items/WYVTD9P2?page=6&a=68YWMV3V)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	Once	no	further	multimodal 	distributions	are	identified,	the	ITS2	sequences	of	the	separated	sets	of	ITS2 sequences	are	considered	 DIVs.	In	this	way,	it	is	possible	 for	different	ITS2	type	profiles	 to	contain	exactly	the	same	collection	of	DIVs	 but	at	different	abundances.	At	this	point,	the	relative	abundances	of	 the ITS2	type	profile's	DIVs	within	all	the	samples	they	were	found	in	 are	then	used	to	characterize	 the	ITS2	type	profile.	Therefore,	when	 searching	for	ITS2	type	profiles	within	samples,	a	match	is	only	considered	if	all	of	the	characterizing	DIVs	are	found,	and	these	have	relative	abundances	 that	fall	within	the	abundance	 ranges	determined	 during	characterization	of	the	given	ITS2	type	profile. 
	
[Page6](zotero://open-pdf/library/items/WYVTD9P2?page=6&a=MPFCXQXH)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	Between	these	genera, 	ITS2	sequences	are	mostly	unalign able.	Therefore,	 as	a	precursor	 to	all	analyses,	 sequence	 data	are	 separated	by	genus/clade	 before	subsequent	analysis	of	ITS2	type	 profiles. 
	
[Page6](zotero://open-pdf/library/items/WYVTD9P2?page=6&a=2YYITE53)
	
	
#symportal
	
	
	reference:

*humeSymPortalNovelAnalytical2019*
	SymPortal	is	able	to	 identify	multiple	ITS2	type	profiles	of	the	same	genus	within	a	given	 sample	as	long	as	each	of	the	 ITS2	type	profiles	 has	been	found	as	 the	sole	 ITS2	type	profile	 in	another	 sample,	such	as	C3-C3a-C3ab	 and	C15-C15x	in	Figure	2	(see	Section	 2.3.6,	SymPortal	output,	for	 an	explanation	of	ITS2	type	profile	systematics). 
	
[Page6](zotero://open-pdf/library/items/WYVTD9P2?page=6&a=HFTMYMX8)
	
	
#symportal
	
	
	reference:


%% end annotations %%

## Bibliography

Hume, B. C. C., Smith, E. G., Ziegler, M., Warrington, H. J. M., Burt, J. A., LaJeunesse, T. C., Wiedenmann, J., & Voolstra, C. R. (2019). SymPortal: A novel analytical framework and platform for coral algal symbiont next‐generation sequencing _ITS2_ profiling. _Molecular Ecology Resources_, _19_(4), 1063–1080. [https://doi.org/10.1111/1755-0998.13004](https://doi.org/10.1111/1755-0998.13004)

%% Import Date: 2023-10-06T10:56:38.898+02:00 %%
