---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[Hyun Jang]], [[Woo Shin]], [[Jeong Lee]], [[Jeong Do]]
collections: [[epigenetic coral]]
Link to zotero:: [Jang et al. - 2017 - CpG and Non-CpG Methylation in Epigenetic Gene Reg.pdf](zotero://select/library/items/SQ66C5CN)
Journal: Genes
DOI: 10.3390/genes8060148
Tags: #epigenetic, #methylation, #üîç

---
```ad-note
title: Abstract
color:: 

DNA methylation is a major epigenetic mark with important roles in genetic regulation. Methylated cytosines are found primarily at CpG dinucleotides, but are also found at non-CpG sites (CpA, CpT, and CpC). The general functions of CpG and non-CpG methylation include gene silencing or activation depending on the methylated regions. CpG and non-CpG methylation are found throughout the whole genome, including repetitive sequences, enhancers, promoters, and gene bodies. Interestingly, however, non-CpG methylation is restricted to speciÔ¨Åc cell types, such as pluripotent stem cells, oocytes, neurons, and glial cells. Thus, accumulation of methylation at non-CpG sites and CpG sites in neurons seems to be involved in development and disease etiology. Here, we provide an overview of CpG and non-CpG methylation and their roles in neurological diseases.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:43 am

*jangCpGNonCpGMethylation2017*
	Methylated cytosines are found primarily at CpG dinucleotides, but are also found at non-CpG sites (CpA, CpT, and CpC). 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=PLZUJBDP)
	
	
#introduction, #methylation
	
	
	reference: ^1015b2

*jangCpGNonCpGMethylation2017*
	The general functions of CpG and non-CpG methylation include gene silencing or activation depending on the methylated regions. 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=QQR2XDTY)
	
	
#introduction, #methylation
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	CpG and non-CpG methylation are found throughout the whole genome, including repetitive sequences, enhancers, promoters, and gene bodies. 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=WHYVUTT9)
	
	
#introduction, #methylation
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	Epigenetic marks include a variety of gene regulatory events, such as chromatin structure remodeling, histone modifications, DNA methylation, and small noncoding RNAs, that do not entail changes in the DNA sequence [1]. 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=Z5PZUQWM)
	
	
#introduction, #epigenetic
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	Gene expression is directly associated with RNA polymerase and transcription factors that bind to regulatory sequence elements, such as promoters and enhancers. Epigenetic events regulate gene expression at both transcription (histone modification and DNA methylation) and translation (small noncoding RNA) levels. 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=XG6QRPBY)
	
	
#introduction, #epigenetic
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	DNA methylation was found to occur predominantly on cytosines followed by guanine residues (CpG). 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=4L9HM5RX)
	
	
#introduction, #epigenetic
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	This type of methylation is referred to as CpG methylation, and cytosine methylated at the fifth carbon of the pyrimidine ring is called 5-methylcytosine (5mC) 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=YBLYW2SR)
	
	
#introduction, #epigenetic
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	The major function of DNA methylation is the suppression of gene expression. Subsequently, DNA methyltransferases (DNMTs), which are specific enzymes that cause methylation at CpG sites, were identified. These enzymes include DNMT1, DNMT3A, and DNMT3B [5]. 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=PW6NAEQR)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference: ^7dd962

*jangCpGNonCpGMethylation2017*
	DNA methylation is also found at sites other than CpG sequences. This type of methylation is referred to as non-CpG methylation, and includes methylation at cytosines followed by adenine, thymine, or another cytosine (Figure 1B) 
	
[Page1](zotero://open-pdf/library/items/SQ66C5CN?page=1&a=6A23XQKF)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	On the other hand, 5mC can also be demethylated via either passive or active DNA demethylation. Passive demethylation is carried out by DNA synthesis without de novo methylation, by which 5mC is diluted following replication. Active demethylation entails the modification of the cytosine base via deamination and/or oxidation [12] (Figure 1C). 
	
[Page2](zotero://open-pdf/library/items/SQ66C5CN?page=2&a=525YML8U)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	Deamination of cytosine by activation induced cytidine deaminase (AID)/apolipoprotein B mRNA editing enzyme, catalytic polypeptide (APOBEC) yields a uracil, which results in a thymine-guanine mismatch, followed by DNA repair-mediated replacement of thymine with unmethylated cytosine. Active demethylation through oxidation process is carried out by Ten-Eleven-Translocation (TET) proteins, including TET1, TET2, and TET3. TETs oxidize the 5-methyl group of cytosine and convert 5mC into 5-hydroxymethylcytosine (5hmC). 5hmC is further oxidized into 5-formylcytosine (5fC) and 5-carboxylcytosine (5caC), which can be excised via glycosylation by thymine DNA glycosylase, followed by base excision repair. This results in unmethylated cytosine [13,14]. Recent studies have revealed that 5hmC is not only an intermediate in the process of active DNA demethylation, but is also a key element in the regulation of gene expression and chromatin structure [15]. 
	
[Page2](zotero://open-pdf/library/items/SQ66C5CN?page=2&a=SLACBZRX)
	
	
#introduction, #epigenetic, #methylation
	
	
	reference:

*jangCpGNonCpGMethylation2017*
	 
	
[Page2](zotero://open-pdf/library/items/SQ66C5CN?page=2&a=8MPXPWFY)
	
	
#introduction, #epigenetic
	
	
	![[Documents/ImageArticles/image-2-x81-y108.png]] 
	
	reference:








%% end annotations %%

## Bibliography

Jang, H., Shin, W., Lee, J., & Do, J. (2017). CpG and Non-CpG Methylation in Epigenetic Gene Regulation and Brain Function. _Genes_, _8_(6), 148. [https://doi.org/10.3390/genes8060148](https://doi.org/10.3390/genes8060148)

%% Import Date: 2023-11-01T14:36:36.022+01:00 %%
