---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[Qian Liu]], [[Li Fang]], [[Guoliang Yu]], [[Depeng Wang]], [[Chuan-Le Xiao]], [[Kai Wang]]
collections: [[epigenetic coral]]
Link to zotero:: [Liu et al. - 2019 - Detection of DNA base modifications by deep recurr.pdf](zotero://select/library/items/HZY9VAUS)
Journal: Nature Communications
DOI: 10.1038/s41467-019-10168-2
Tags: #epigenetic, #methylation, #nanopore, #üìù, #üîç, #üé®

---
```ad-note
title: Abstract
color:: 

Abstract
            
              DNA base modifications, such as C5-methylcytosine (5mC) and N6-methyldeoxyadenosine (6mA), are important types of epigenetic regulations. Short-read bisulfite sequencing and long-read PacBio sequencing have inherent limitations to detect DNA modifications. Here, using raw electric signals of Oxford Nanopore long-read sequencing data, we design DeepMod, a bidirectional recurrent neural network (RNN) with long short-term memory (LSTM) to detect DNA modifications. We sequence a human genome HX1 and a
              Chlamydomonas reinhardtii
              genome using Nanopore sequencing, and then evaluate DeepMod on three types of genomes (
              Escherichia coli
              ,
              Chlamydomonas reinhardtii
              and human genomes). For 5mC detection, DeepMod achieves average precision up to 0.99 for both synthetically introduced and naturally occurring modifications. For 6mA detection, DeepMod achieves ~0.9 average precision on
              Escherichia coli
              data, and have improved performance than existing methods on
              Chlamydomonas reinhardtii
              data. In conclusion, DeepMod performs well for genome-scale detection of DNA modifications and will facilitate epigenetic analysis on diverse species.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:43 am

*liuDetectionDNABase2019*
	For 5mC detection, DeepMod achieves average precision up to 0.99 for both synthetically introduced and naturally occurring modifications. For 6mA detection, DeepMod achieves ~0.9 average precision on Escherichia coli data, and have improved performance than existing methods on Chlamydomonas reinhardtii data. 
	
[Page1](zotero://open-pdf/library/items/HZY9VAUS?page=1&a=W3E3XQGB)
	
	
#methodology, #methylation, #nanopore
	
	
	reference:

*liuDetectionDNABase2019*
	DNA methylations, including 5-methylcytosine (5mC), 5-hydroxymethylcytosine (5hmC), and N6-methyldeoxyadenosine (6mA), are introduced into a DNA molecule by adding methyl or hydroxymethyl groups to nucleotides: 5mC is introduced by adding methyl group at the 5th position of the pyrimidine ring of cytosines, whereas 6mA is introduced by adding methyl group at the 6th position of the purine ring in adenines. 
	
[Page2](zotero://open-pdf/library/items/HZY9VAUS?page=2&a=NYWN6H3G)
	
	
#introduction, #methylation
	
	
	reference: ^0d14e6

*liuDetectionDNABase2019*
	DNA base modifications widely exist in different organisms3 and are essential in various biological processes4,5, such as genomic imprinting, X-chromosome inactivation, genome stability, gene regulation, repression of transposable elements, aging, and carcinogenesis. 
	
[Page2](zotero://open-pdf/library/items/HZY9VAUS?page=2&a=LKVLJEJX)
	
	
#introduction, #methylation
	
	
	reference: ^b1207f

*liuDetectionDNABase2019*
	peutic target. DNA modifications, especially DNA methylations, could be detected by both short-read sequencing and long-read sequencing techniques. Bisulfite sequencing with short-read techniques is widely used to call methylated cytosines9 by converting unmethylated cytosines to uracil, but bisulfite sequencing and its improved variations10 were limited by the efficiency in conversion and the inability to assay repetitive genomic regions by short-read sequencing. 
	
[Page2](zotero://open-pdf/library/items/HZY9VAUS?page=2&a=JXJHBCST)
	
	
#methodology, #methylation, #WGBS
	
	
	reference:

*liuDetectionDNABase2019*
	Recently, several proof-of-concept studies on Oxford Nanopore sequencing techniques have demonstrated the feasibility to detect DNA modifications based on the electric signal characteristics when a modified DNA molecule passes through nanopores22,23. 
	
[Page2](zotero://open-pdf/library/items/HZY9VAUS?page=2&a=MJZ9MRGF)
	
	
#methodology, #methylation, #nanopore
	
	
	reference:








%% end annotations %%

## Bibliography

Liu, Q., Fang, L., Yu, G., Wang, D., Xiao, C.-L., & Wang, K. (2019). Detection of DNA base modifications by deep recurrent neural network on Oxford Nanopore sequencing data. _Nature Communications_, _10_(1), 2449. [https://doi.org/10.1038/s41467-019-10168-2](https://doi.org/10.1038/s41467-019-10168-2)

%% Import Date: 2023-11-01T14:36:40.709+01:00 %%
