---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[Kim Pham]], [[Long Ho]], [[Claudio Peter D'Incal]], [[Andrée De Cock]], [[Wim Vanden Berghe]], [[Peter Goethals]]
collections: [[epigenetic coral]]
Link to zotero:: 
Journal: Environmental Pollution
DOI: 10.1016/j.envpol.2023.121737
Tags: #❌, #Aquatic animal, #DNA methylation, #environmental Epigenetics, #Environmental stress, #Histone modification, #Pollution

---
```ad-note
title: Abstract
color:: 

Environmental epigenetics has become a key research focus in global climate change studies and environmental pollutant investigations impacting aquatic ecosystems. Specifically, triggered by environmental stress conditions, intergenerational DNA methylation changes contribute to biological adaptive responses and survival of organisms to increase their tolerance towards these conditions. To critically review epigenetic analytical approaches in ecotoxicological aquatic research, we evaluated 78 publications reported over the past five years (2016–2021) that applied these methods to investigate the responses of aquatic organisms to environmental changes and pollution. The results show that DNA methylation appears to be the most robust epigenetic regulatory mark studied in aquatic animals. As such, multiple DNA methylation analysis methods have been developed in aquatic organisms, including enzyme restriction digestion-based and methyl-specific immunoprecipitation methods, and bisulfite (in)dependent sequencing strategies. In contrast, only a handful of aquatic studies, i.e. about 15%, have been focusing on histone variants and post-translational modifications due to the lack of species-specific affinity based immunological reagents, such as specific antibodies for chromatin immunoprecipitation applications. Similarly, ncRNA regulation remains as the least popular method used in the field of environmental epigenetics. Insights into the opportunities and challenges of the DNA methylation and histone variant analysis methods as well as decreasing costs of next generation sequencing approaches suggest that large-scale epigenetic environmental studies in model and non-model organisms will soon become available in the near future. Moreover, antibody-dependent and independent methods, such as mass spectrometry-based methods, can be used as an alternative epigenetic approach to characterize global changes of chromatin histone modifications in future aquatic research. Finally, a systematic guide for DNA methylation and histone variant methods is offered for ecotoxicological aquatic researchers to select the most relevant epigenetic analytical approach in their research.

```

---
### Highlight

%% begin annotations %%



%% end annotations %%

## Bibliography

Pham, K., Ho, L., D’Incal, C. P., De Cock, A., Berghe, W. V., & Goethals, P. (2023). Epigenetic analytical approaches in ecotoxicological aquatic research. _Environmental Pollution_, _330_, 121737. [https://doi.org/10.1016/j.envpol.2023.121737](https://doi.org/10.1016/j.envpol.2023.121737)

%% Import Date: 2023-11-01T14:36:49.446+01:00 %%
