---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[Javier A. Rodr√≠guez-Casariego]], [[Alex E. Mercado-Molina]], [[Daniel Garcia-Souto]], [[Ivanna M. Ortiz-Rivera]], [[Christian Lopes]], [[Iliana B. Baums]], [[Alberto M. Sabat]], [[Jose M. Eirin-Lopez]]
collections: [[epigenetic coral]]
Link to zotero:: [Rodr√≠guez-Casariego et al. - 2020 - Genome-Wide DNA Methylation Analysis Reveals a Con.pdf](zotero://select/library/items/8V7UFISV)
Journal: Frontiers in Marine Science
DOI: 10.3389/fmars.2020.560424
Tags: #üîç, #üé®

---
```ad-note
title: Abstract
color:: 

Epigenetic modiÔ¨Åcations such as DNA methylation have been shown to participate in plastic responses to environmental change in a wide range of organisms, including scleractinian corals. Unfortunately, the current understanding of the links between environmental signals, epigenetic modiÔ¨Åcations, and the subsequent consequences for acclimatory phenotypic changes remain obscure. Such a knowledge gap extends also to the dynamic nature of epigenetic changes, hampering our ability to ascertain the magnitude and extent of these responses under natural conditions. The present work aims to shed light on these subjects by examining temporal changes in genomewide patterns of DNA methylation in the staghorn coral Acropora cervicornis in the island of Culebra, PR. During a 17-month period, a total of 162 polymorphic loci were identiÔ¨Åed using Methylation-Sensitive AmpliÔ¨Åed Polymorphism (MSAP). Among them, 83 of these restriction fragments displayed changes in DNA methylation that were signiÔ¨Åcantly correlated to seasonal variation as determined mostly by changes in sea water temperature. Remarkably, the observed time-dependent variation in DNA methylation patterns is consistent across coral genets, coral source sites and sitespeciÔ¨Åc conditions studied. Overall, these results are consistent with a conserved epigenetic response to seasonal environmental variation. These Ô¨Åndings highlight the importance of including seasonal variability into experimental designs investigating the role of epigenetic mechanisms such as DNA methylation in responses to stress.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:43 am

*rodriguez-casariegoGenomeWideDNAMethylation2020*
	Genome-wide DNA methylation was assessed on coral hostenriched-DNA samples using an amplified polymorphism approach specific for DNA methylation states (Methylation Sensitive Amplified Polymorphism, MSAP 
	
[Page5](zotero://open-pdf/library/items/8V7UFISV?page=5&a=XENA7XBB)
	
	
#methodology, #epigenetic, #GDM
	
	
	reference:

*rodriguez-casariegoGenomeWideDNAMethylation2020*
	This method is based on the use of isoesquizomeric endonucleases, HpaII and MspI, with shared sequence targets (CCGG sites) but differential sensitivities to their DNA methylation. More precisely, HpaII cleavage is blocked by either internal cytosine methylation of the target site (i.e., 5‚Ä≤-CmCGG-3‚Ä≤/3‚Ä≤-GGmCC-5‚Ä≤) or its hypermethylation (i.e., 5‚Ä≤-mCmCGG-3‚Ä≤/3‚Ä≤-GGmCmC-5‚Ä≤). MspI, on the other hand, is sensitive to external cytosine methylation, including hemimethylation (i.e., 5‚Ä≤-mCCGG-3‚Ä≤/3‚Ä≤-GGCC-5‚Ä≤) and hypermethylation states. This allows the establishment of global cytosine methylation patterns by comparing both amplified restriction profiles (D√≠az-Freije et al., 2014). Accordingly, coral genomic DNA was digested using EcoRI/HpaII and EcoRI/MspI endonuclease mixes in parallel reactions. In the same step, the resulting fragments were ligated to EcoRI and HpaII/MspI adapters (Table 1). Digestion-ligation reactions were performed for 2 h at 37‚ó¶C in a solution consisting of 200 ng DNA, 4 U of EcoRI (NEB, Ipswich, MA, United States), 1 U of either HpaII (NEB, Ipswich, MA, United States) or MspI (NEB, Ipswich, MA, United States), 1 U T4 DNA ligase (Thermo Fisher Scientific, Waltham, MA, United States), 1X ligase buffer (Thermo Fisher Scientific, Waltham, MA, United States) and 1X CutSmart Buffer (NEB, Ipswich, MA, United States). The resulting restriction fragments were selectively amplified through two consecutive PCR reactions. First, a pre-selective reaction containing 2 ŒºL of diluted (1:7) restriction-ligation product, 20 pM of each HpaII/MspI and EcoRI primers combination (Table 1), 1X PCR buffer, 0.5 mM dNTPs (Thermo Fisher Scientific, Waltham, MA, United States), 2.5 mM MgCl2, and 1 U DreamTAQ DNA polymerase (Thermo Fisher Scientific, Waltham, MA, United States). Second, a selective reaction used 0.5 ŒºL of 1:9 of the pre-selective PCR product, 0.83 pM of each labeled selective primer (Table 1), 1X PCR buffer, 0.5 mM dNTPs, 2.5 mM MgCl2 and 1 U DreamTaq DNA polymerase. PCR conditions were identical to the original protocol (Reyna-Lopez et al., 1997), and the amplified products (2 per enzyme/sample combination, 4 selective combinations multiplexed, Table 1) were diluted to 1:10 for 6-FAM and 1:5 for 6-HEX prior to multiplexing and run on an ABI Prism 310 Genetic Analyzer (Applied Biosystems, Foster City, CA, United States) with a MapMarker 1000 ROX marker at Florida International University‚Äôs DNA Core facility. 
	
[Page5](zotero://open-pdf/library/items/8V7UFISV?page=5&a=HX3U5L8C)
	
	
#methodology, #epigenetic, #GDM
	
	
	reference:








### Imported: 2023-10-11 5:00 pm

*rodriguez-casariegoGenomeWideDNAMethylation2020*
	Contrary to the case of random environmental variation, the predictability of seasonal fluctuations can be conducive to the development and inheritance of plastic transcriptional profiles mediating phenotypic responses, regulated by epigenetic mechanisms [e.g., seasonal DNA methylation changes in bivalve mollusks (SuarezUlloa et al., 2019), plants (Ito et al., 2019) and birds (Viitaniemi et al., 2019)] 
	
[Page2](zotero://open-pdf/library/items/8V7UFISV?page=2&a=X5IQZ63F)
	
	
#DNA-Methylation, #epigenetic, #introduction
	
	
	reference:

*rodriguez-casariegoGenomeWideDNAMethylation2020*
	Indeed, seasonality produces dramatic physiological adjustments in corals, including changes in symbiont‚Äôs abundance and pigmentation (Fitt et al., 2000; Thornhill et al., 2006), modifications of microbial community composition (Sharp et al., 2017), as well as the alteration of transcriptional profiles (Edge et al., 2008; Brady et al., 2011; Parkinson et al., 2018; BrenerRaffalli et al., 2019). 
	
[Page2](zotero://open-pdf/library/items/8V7UFISV?page=2&a=K5WBALNV)
	
	
#epigenetic, #introduction, #microbiome, #transcriptomic
	
	
	reference: ^b388e6






### Imported: 2023-11-01 2:32 pm

*rodriguez-casariegoGenomeWideDNAMethylation2020*
	Among them, 83 of these restriction fragments displayed changes in DNA methylation that were significantly correlated to seasonal variation as determined mostly by changes in sea water temperature. Remarkably, the observed time-dependent variation in DNA methylation patterns is consistent across coral genets, coral source sites and sitespecific conditions studied. Overall, these results are consistent with a conserved epigenetic response to seasonal environmental variation. 
	
[Page1](zotero://open-pdf/library/items/8V7UFISV?page=1&a=7MCIL45A)
	
	
#introduction, #Discussion, #epigenetic, #methylation, #seasonalenvironmentalvariation
	
	
	reference:


%% end annotations %%

## Bibliography

Rodr√≠guez-Casariego, J. A., Mercado-Molina, A. E., Garcia-Souto, D., Ortiz-Rivera, I. M., Lopes, C., Baums, I. B., Sabat, A. M., & Eirin-Lopez, J. M. (2020). Genome-Wide DNA Methylation Analysis Reveals a Conserved Epigenetic Response to Seasonal Environmental Variation in the Staghorn Coral _Acropora cervicornis_. _Frontiers in Marine Science_, _7_, 560424. [https://doi.org/10.3389/fmars.2020.560424](https://doi.org/10.3389/fmars.2020.560424)

%% Import Date: 2023-11-01T14:36:53.436+01:00 %%
