---
Used: false
collections: "[[epigenetic coral]]"
tipe: article
---
tipe: journalArticle
Creators: [[William Walters]], [[Embriette R. Hyde]], [[Donna Berg-Lyons]], [[Gail Ackermann]], [[Greg Humphrey]], [[Alma Parada]], [[Jack A. Gilbert]], [[Janet K. Jansson]], [[J. Gregory Caporaso]], [[Jed A. Fuhrman]], [[Amy Apprill]], [[Rob Knight]]
collections: [[epigenetic coral]]
Link to zotero:: [Walters et al_2016_Improved Bacterial 16S rRNA Gene (V4 and V4-5) and Fungal Internal Transcribed.pdf](zotero://select/library/items/PG3H53JQ)
Journal: mSystems
DOI: 10.1128/mSystems.00009-15
Tags: #‚ùå, #üìù, #16S, #ITS, #marker genes, #microbial ecology, #primers

---
```ad-note
title: Abstract
color:: 

Designing primers for PCR-based taxonomic surveys that amplify a broad range of phylotypes in varied community samples is a difficult challenge, and the comparability of data sets amplified with varied primers requires attention. Here, we examined the performance of modified 16S rRNA gene and internal transcribed spacer (ITS) primers for archaea/bacteria and fungi, respectively, with nonaquatic samples. We moved primer bar codes to the 5' end, allowing for a range of different 3' primer pairings, such as the 515f/926r primer pair, which amplifies variable regions 4 and 5 of the 16S rRNA gene. We additionally demonstrated that modifications to the 515f/806r (variable region 4) 16S primer pair, which improves detection of Thaumarchaeota and clade SAR11 in marine samples, do not degrade performance on taxa already amplified effectively by the original primer set. Alterations to the fungal ITS primers did result in differential but overall improved performance compared to the original primers. In both cases, the improved primers should be widely adopted for amplicon studies. IMPORTANCE We continue to uncover a wealth of information connecting microbes in important ways to human and environmental ecology. As our scientific knowledge and technical abilities improve, the tools used for microbiome surveys can be modified to improve the accuracy of our techniques, ensuring that we can continue to identify groundbreaking connections between microbes and the ecosystems they populate, from ice caps to the human body. It is important to confirm that modifications to these tools do not cause new, detrimental biases that would inhibit the field rather than continue to move it forward. We therefore demonstrated that two recently modified primer pairs that target taxonomically discriminatory regions of bacterial and fungal genomic DNA do not introduce new biases when used on a variety of sample types, from soil to human skin. This confirms the utility of these primers for maintaining currently recommended microbiome research techniques as the state of the art.

```

---
### Highlight

%% begin annotations %%



### Imported: 2023-10-06 10:43 am

*waltersImprovedBacterial16S2016*
	We additionally demonstrated that modifications to the 515f/806r (variable region 4) 16S primer pair, which improves detection of Thaumarchaeota and clade SAR11 in marine samples, do not degrade performance on taxa already amplified effectively by the original primer set. 
	
[Page1](zotero://open-pdf/library/items/PG3H53JQ?page=1&a=YIJDUDA5)
	
	
#methodology, #microbiome
	
	
	reference:

*waltersImprovedBacterial16S2016*
	S urveys of the small ribosomal subunits (SSUs) for Bacteria, Archaea, and Eukarya and the internal transcribed spacer (ITS) region for fungi are central to microbial ecology. 
	
[Page2](zotero://open-pdf/library/items/PG3H53JQ?page=2&a=5Y6V9GYB)
	
	
#methodology, #microbiome
	
	
	reference:

*waltersImprovedBacterial16S2016*
	we tested modified forms of the 515f and 806r 16S rRNA gene (variable region 4) primers 
	
[Page2](zotero://open-pdf/library/items/PG3H53JQ?page=2&a=MJK3KI8N)
	
	Variable region 4 (often referred to as V4) is one of the hypervariable regions of the 16S rRNA gene in bacteria and archaea. The 16S rRNA gene is commonly used in microbial ecology and microbiome studies to identify and classify bacteria and archaea because it is present in almost all bacteria and archaea, but has regions of sequence variability that can be used to distinguish between different species or strains.
	
	
#methodology, #microbiome, #16S
	
	
	reference:

*waltersImprovedBacterial16S2016*
	For fungal surveys, the ITS1F (6) and ITS2 primer pair (7) has been commonly used. 
	
[Page2](zotero://open-pdf/library/items/PG3H53JQ?page=2&a=M6YEXFAZ)
	
	
#methodology, #microbiome, #ITS, #fungal
	
	
	reference:

*waltersImprovedBacterial16S2016*
	we compared the new and old constructs for both 16S rRNA and ITS marker genes, and we also compared the performance of the modified 515f/806rB (V4) primer constructs to the longer 515f/926r (V4-5) constructs. 
	
[Page2](zotero://open-pdf/library/items/PG3H53JQ?page=2&a=2IL4BGCZ)
	
	
#methodology, #microbiome
	
	
	reference:

*waltersImprovedBacterial16S2016*
	Here, we showed that the 515f/806rB bacterial/archaeal and ITS fungal primer pairs do not introduce detrimental biases compared to the original constructs in nonaquatic environments. The modified 515f/806rB results were highly concordant with the original 515f/806r and 515f/926r results. 
	
[Page7](zotero://open-pdf/library/items/PG3H53JQ?page=7&a=BE4F4GQX)
	
	
#methodology, #microbiome
	
	
	reference:








%% end annotations %%

## Bibliography

Walters, W., Hyde, E. R., Berg-Lyons, D., Ackermann, G., Humphrey, G., Parada, A., Gilbert, J. A., Jansson, J. K., Caporaso, J. G., Fuhrman, J. A., Apprill, A., & Knight, R. (2016). Improved Bacterial 16S rRNA Gene (V4 and V4-5) and Fungal Internal Transcribed Spacer Marker Gene Primers for Microbial Community Surveys. _mSystems_, _1_(1), e00009-15. [https://doi.org/10.1128/mSystems.00009-15](https://doi.org/10.1128/mSystems.00009-15)

%% Import Date: 2023-11-01T14:37:03.398+01:00 %%
