---
Done: false
"date:": 
tags: 
tipe: article
---
# Data analysis 
[[Delimitation species ]]

# Locus 
![[paste.png]]
## ITS2 + 28 S
### PCR

ITSc2-5: 5'-AGCCAGCTGCGATAAGTAGTG-3' 
R28S1: 5'-GCTGCAATCCCAAACAACCC-3'

### Sequencing sanger

ITSc2-5: 5'-AGCCAGCTGCGATAAGTAGTG-3' 
R28S1: 5'-GCTGCAATCCCAAACAACCC-3'

## ATPSbeta 

### PCR 

ATPSβf5: 5'-CCAAGGGTGGNAARATHGGT-3' 
ATPSβr2: 5'-GGTTCGTTCATCTGACCATACAC-3'

### Sequencing sanger

ATPSβf2: 5'-TGAAAGACAAGAGCTCCAAGGTA-3'
ATPSβf4: 5'-GAGCTGGTGTTGGAAAGACTGT-3'

## ORF 



## CR 



# Result for the locus 
## ITS2 Cliperton + Mexique 

![[paste (1).png]]
cliperton = red 
Mexique = green 

- [x] rexplain how it works #qst

## ATPSbeta Cliperton + Mexique

![[paste (2).png]]

cliperton = red 
Mexique = green 

- [ ] instal haplowebmaker #todo  Spöri Y, Flot J (2020) HaplowebMaker and CoMa: two web tools to delimit species using haplowebs and conspecificity matrices. Methods in Ecology and Evolution 11:1434–1438


# Matrice de conspecificité 

![[paste (3).png]]

- [x] Whats is that #qst

# Article d'interet a lire 

- Dellicour S, Flot J-F (2018) The hitchhiker’s guide to single-locus species delimitation. Molecular Ecology Resources 18:1234–1246

# Protocol 
## set primers PCR1

| gene     | primers    |
| -------- | ---------- |
| ITS2     | ITSc2      |
|          | R28S1      |
| -----    | -----      |
| ATPSbeta | ATPSbetar2 |
|          | ATPSbetaf5 |
| ------   | -----      |
| ORF      | FATP6.1    |
|          | RORF       |
| -----    | ------     |
| CR       | FNAD5.2deg |
|          | RCOI3deg   |


## Cycle 
cycle : 45
red taq 


| time  | temp |
| ----- | ---- |
| 60 s  | 94°c |
| 30 s  | 94°C |
| 30 s  | 53°C |
| 75 s  | 72°C |
| 5 min | 72°C |


hello 
